test/e2e/whoami.e2e.js

```diff
@@ -1,9 +1,10 @@
+'use strict';
 const { expect } = require('../setup');
 const cli = require('../lib/cli');
 const {
 	USERNAME
 } = require('../lib/env');
-const stripAnsi = require('strip-ansi');
+const { default: stripAnsi } = require('strip-ansi');
```

Comment:
As far I see is jut stripAnsi the problem maybe? if so disregard this and the comment bellow 😄